src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+
+        <div class="element-form">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </div>
+
+        <div class="element-form">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name:"
```

Comment:
Не додавай до назви інпута двокрапку. Загалом, навза може співпадати з айдішкою
```suggestion
            name="name"
```

Reply:
дякую, не замітив що в деяких місцях додав двокрапку! Вже виправив!